component-maps.yml

```diff
@@ -234,6 +236,13 @@ docker_image:
     - mender-client
     release_component: true
 
+  mender-device-docker:
+    git:
+    - integration
+    docker_container:
+    - mender-client
+    release_component: true
```

Comment:
Ah, hehe, now I understand why you had concerns about the release_tool! But I *think* it should work. Logically it makes sense, at least.

But should it have entries in `docker_container` and `git` sections as well..?

Reply:
> Ah, hehe, now I understand why you had concerns about the release_tool! But I think it should work. Logically it makes sense, at least.

I also *think* it should work, but let's be aware during next release...

> But should it have entries in docker_container and git sections as well..?

Yes. Aren't they there? For `git` in under `integration`, for `docker_container` is under `mender-client`. I think it is correct, but please check again.